src/lib/actions/RulesAction.ts

```diff
@@ -33,6 +33,7 @@ export class RulesAction {
 	public async execute(input: RulesInput): Promise<void> {
 		const config: CodeAnalyzerConfig = this.dependencies.configFactory.create(input['config-file']);
 		const logWriter: LogFileWriter = await LogFileWriter.fromConfig(config);
+		this.dependencies.actionSummaryViewer.viewPreExecutionSummary(logWriter.getLogDestination());
```

Comment:
This is the functional change to this Action. I added it as early in the method as I could (i.e., immediately after the log file writer is instantiated) to minimize the possibility of it being preempted by an error.
If we want, though, we could easily move it further down and use it in the future to log additional information about, for example, the engines that are added. I'm flexible on this.